robolectric/src/main/java/org/robolectric/internal/dependency/LocalDependencyResolver.java

```diff
@@ -8,9 +8,11 @@
 
 public class LocalDependencyResolver implements DependencyResolver {
   private File offlineJarDir;
+  private final DependencyResolver dependencyResolver;
 
-  public LocalDependencyResolver(File offlineJarDir) {
+  public LocalDependencyResolver(DependencyResolver dependencyResolver, File offlineJarDir) {
```

Comment:
What about CachedDependencyResolver? It already receives the MavenDependencyResolver on its constructor. I can try to change it, but how will the CachedDependencyResolver find out about the resolved dependency in order to cache it?